src/odemis/dataio/csv.py

```diff
@@ -62,13 +63,26 @@ def export(filename, data):
             csv_writer = csv.writer(fd)
             csv_writer.writerow(headers)
             csv_writer.writerows(spectrum_tuples)
-    elif data.ndim == 2 and all(s >= 2 for s in data.shape):
+    elif data.ndim == 2 and all(s >= 2 for s in data.shape) and (data.metadata.get(model.MD_DESCRIPTION, None) == "Angle-resolved"):
```

Comment:
I like the MD_ACQ_TYPE option, should I try to update all the drivers to include this in their acquisition?